watertap/unit_models/gac.py

```diff
@@ -270,475 +299,394 @@ def build(self):
-            units=units_meta("mass"),
-            doc="Mass of contaminant adsorbed if fully saturated",
+            units=pyunits.dimensionless,
+            doc="Biot number",
         )
 
         # ---------------------------------------------------------------------
-        # gac bed properties
+        # bed dimensions and related variables
+
+        self.velocity_sup = Var(
+            initialize=0.001,
+            bounds=(0, None),
+            domain=NonNegativeReals,
+            units=units_meta("length") * units_meta("time") ** -1,
+            doc="superficial velocity",
+        )
+        self.velocity_int = Var(
+            initialize=0.002,
+            bounds=(0, None),
+            domain=NonNegativeReals,
+            units=units_meta("length") * units_meta("time") ** -1,
+            doc="interstitial velocity",
+        )
         self.bed_voidage = Var(
             initialize=0.5,
             bounds=(0, 1),
             domain=NonNegativeReals,
             units=pyunits.dimensionless,
-            doc="Adsorber bed void fraction",
-        )
-
-        self.bed_volume = Var(
-            initialize=100,
-            bounds=(1e-3, None),
-            domain=NonNegativeReals,
-            units=units_meta("length") ** 3,
-            doc="Adsorber bed volume",
+            doc="bed void fraction",
         )
-
-        self.bed_area = Var(
-            initialize=1,
+        self.bed_length = Var(
+            initialize=5,
             bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("length") ** 2,
-            doc="Adsorber bed area, single adsorber area or sum of areas for adsorbers in parallel",
+            units=units_meta("length"),
+            doc="bed length",
         )
-
         self.bed_diameter = Var(
             initialize=1,
             bounds=(0, None),
             domain=NonNegativeReals,
             units=units_meta("length"),
-            doc="Adsorber bed diameter, valid if considering only a single adsorber",
+            doc="bed diameter, valid if considering only a single adsorber",
         )
-
-        self.bed_length = Var(
+        self.bed_area = Var(
             initialize=1,
             bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("length"),
-            doc="Adsorber bed length",
+            units=units_meta("length") ** 2,
+            doc="bed area, single adsorber area or sum of areas for adsorbers in parallel",
         )
-
-        self.bed_mass_gac = Var(
-            initialize=100,
+        self.bed_volume = Var(
+            initialize=5,
             bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("mass"),
-            doc="Mass of fresh GAC in the adsorber bed",
+            units=units_meta("length") ** 3,
+            doc="bed volume",
         )
-
-        self.velocity_sup = Var(
-            initialize=1,
+        self.ebct = Var(
```

Comment:
not going to be picky with naming conventions in this PR, but it is generally preferable to be explicit and spell names out for better readability for users (albeit we might expect a user in this field of research to be familiar with ebct).

Reply:
I used this sparingly for EBCT and SPDFR just because the black formatting when using min_empty_bed_contact_time and surface_to_pore_diffusion_flux_ratio gets pretty illegible because of the length. I think for EBCT, it's such a fundamental concept in the field, like having to write out plug flow reactor, that I would think it's fine. I'm not opposed to changing it to maintain a convention though.